var/spack/repos/builtin/packages/py-ipykernel/package.py

```diff
@@ -42,15 +42,10 @@ class PyIpykernel(Package):
     version('4.1.1', '51376850c46fb006e1f8d1cd353507c5')
     version('4.1.0', '638a43e4f8a15872f749090c3f0827b6')
 
-    extends('python')
-
     depends_on('python@2.7:2.7.999,3.3:')
```

Comment:
This won't work. There is no Python version 2.7, so it will never be matched. Try editing it and running `spack spec`.